src/isomorphic/modern/class/ReactBaseClasses.js

```diff
@@ -21,7 +21,28 @@ var warning = require('fbjs/lib/warning');
 /**
  * Base class helpers for the updating state of a component.
  */
+
+const addUpdaterMutationWarn = context => {
```

Comment:
So that means that we need a `// eslint-disable-next-line no-undef` every time `addUpdaterMutationWarn` is used right?